frontend/components/NavDrawerCategoriesTree/components/CategoriesItem/index.jsx

```diff
@@ -78,6 +94,12 @@ const CategoriesItem = ({
     setIsOpen(isPartOfCategoryPath);
   }, [isPartOfCategoryPath]);
 
+  useEffect(() => {
```

Comment:
when i navigate to one of the screens from the page-switcher extension, the correct category is selected and the subcategories are visible.
when i now navigate to a sub-category and press the back button afterwards, the category tree is closed.

Reply:
I added `activeCategoryPath` as a dependent variable for `useEffect`, so that the effect will run again with a change of the active path.